src/Core/Models/Business/OrganizationUpdate.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Bit.Core.Enums;
+
+namespace Bit.Core.Models.Business;
+
+public class OrganizationUpdate
+{
+    public Guid OrganizationId { get; set; }
+    public int SeatAdjustment { get; set; }
+    public int? MaxAutoscaleSeats { get; set; }
+
+    public int? ServiceAccountsAdjustment { get; set; }
+    public int? MaxAutoscaleServiceAccounts { get; set; }
```

Comment:
You remove null values in the request model's `ToOrganizationUpdate` method by calling `GetValueOrDefault`. So these don't need to be nullable here.